webapp/src/app.tsx

```diff
@@ -1168,6 +1168,42 @@ export class ProjectView extends data.Component<IAppProps, IAppState> {
             })
     }
 
+    launchFullEditor() {
+        let sandbox = /sandbox=1/i.test(window.location.href)
+        let sandboxPub = /#sandbox:/i.test(window.location.href)
+        let sandboxProject = /#sandboxproject:/i.test(window.location.href)
+        Util.assert(sandbox || sandboxPub || sandboxProject);
```

Comment:
this assertion is invalid. we do enter the sandbox mode automatically when hosted in iframe. See global sandbox flag.